packages/building-rollup/test-node/integration.test.js

```diff
@@ -9,7 +8,13 @@ const { startDevServer } = require('@web/dev-server');
 
 const rootDir = path.resolve(__dirname, '..', 'dist');
 
-describe('integration tests', () => {
+describe('integration tests', async () => {
+  let expect;
```

Comment:
cant we just

```js
const { expect } = await import('chai');
```

instead of doing it in a `before` block?

Reply:
I was doing that before but had to change to this because I was getting silent failures in mocha. Some googling around brought me to https://github.com/mochajs/mocha/issues/2975#issuecomment-355282416 which seemed to indicate that wrapping it in a `before` would fix the issue. And it seems to fix it for me.

Reply:
wow TIL, thats wild

Reply:
It's a bit weird. This can then all be removed once this is all ESM.